### PR TITLE
Mg webpage redesign

### PR DESCRIPTION
**Pull request by mahdisghaffari**
first design by mahdis

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,8 +13,8 @@
 <body class="fullscreen">
 	<div class="container" id="id-table-search">
 		<h1 class="title">Timetrack</h1>
-		<input id="task-name" type="text" value="" placeholder="Task name" autofocus/>
-		<input id="task-description" type="text" value="" placeholder="Task description"/>
+                <input id="task-name" class="form-control" type="text" value="" placeholder="Task name" autofocus/>
```
Comment (amiartus) [inline index.html]: please fix whitespace error

File: js/main.js [hunk 2]
```diff
@@ -314,11 +314,14 @@ async function update_task_ui(task_name)
 	task = await db_get(table_running, 1);
 	// if task is running allow stop
 	if (task != undefined && task[key_name] == task_name) {
-		button.addEventListener("click", on_task_stop);
-		button.value = "stop";
+                button.addEventListener("click", on_task_stop);
```
Comment (amiartus) [inline js/main.js]: please use tabs

File: js/main.js [hunk 3]
```diff
@@ -247,9 +247,10 @@ async function on_task_start()
 	// start a task
 	let task_name = document.getElementById("task-name").value;
 	if (task_name != undefined) {
+                let button = document.getElementById("task-update");
```
Comment (amiartus) [inline js/main.js]: tabs

File: index.html [hunk 4]
```diff
@@ -4,10 +4,14 @@
 <head>
 	<meta charset="utf-8">
 	<title>timetrack</title>
+                <link href="https://fonts.googlseapis.com/css2?family=Anton&family=Chelsea+Market&display=swap" rel="stylesheet">
```
Comment (amiartus) [inline index.html]: tabs

File: css/style.css [hunk 5]
```diff
@@ -40,6 +40,7 @@ input[type="button"] {
 
 .title {
 	font-size: 3rem;
+font-family: 'Lobster', cursive;
```
Comment (amiartus) [inline css/style.css]: whitespace

File: css/style.css [hunk 6]
```diff
@@ -46,13 +46,61 @@ font-family: 'Lobster', cursive;
 .table {
 	margin: auto;
 	width: 100%;
-	border: 1px solid black;
+/*	border: 1px solid black;*/
```
Comment (amiartus) [inline css/style.css]: remove commented lines

File: css/style.css [hunk 7]
```diff
@@ -70,24 +70,24 @@ font-family: 'Lobster', cursive;
   padding: 5px 15px;
 }
 .table td {
-  border-width: 3px 0;
+  /*border-width: 3px 0;*/
```
Comment (amiartus) [inline css/style.css]: remove commented lines

File: index.html (hunk 1 above)
Comment (mahdisghaffari) [reply index.html]: I fixed!!!!

**Pull request merged**